client/src/utils/notificationUtils.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { notification } from 'antd';
+import { CheckCircleOutlined, CloseCircleOutlined, CloudUploadOutlined, WarningOutlined } from '@ant-design/icons';
+import corporateDesign from '../layout/corporateDesign';
+
+/**
+ * @description will first consider if a special icon is requested and otherwise return the corresponding icon for the notification type.
```

Comment:
Please start the description with a capital letter, for parameters its fine, but here please fix this

Reply:
is fixed